icons/k6/k6-original.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px"
+  viewBox="0 0 128 128" version="1.1">
+  <g id="surface1">
+    <path style=" stroke:none;fill-rule:nonzero;fill:rgb(49.019608%,39.215686%,100%);fill-opacity:1;"
+      d="M 127.828125 128 L 0.171875 128 L 42.667969 34.34375 L 68.269531 53.785156 L 101.660156 0 Z M 81.023438 107.769531 L 81.296875 107.769531 C 84.457031 107.777344 87.496094 106.515625 89.769531 104.25 C 90.9375 103.15625 91.863281 101.820312 92.492188 100.328125 C 93.121094 98.835938 93.433594 97.222656 93.414062 95.59375 C 93.464844 94.027344 93.175781 92.46875 92.570312 91.03125 C 91.960938 89.59375 91.050781 88.3125 89.902344 87.289062 C 87.96875 85.28125 85.359375 84.121094 82.617188 84.046875 L 82.40625 84.046875 C 82.046875 84.042969 81.691406 84.09375 81.347656 84.195312 L 88.085938 73.847656 L 82.71875 69.976562 L 80.175781 73.847656 L 73.683594 84.09375 C 72.566406 85.792969 71.632812 87.289062 71.054688 88.328125 C 70.457031 89.425781 69.941406 90.574219 69.511719 91.757812 C 69.027344 93.011719 68.773438 94.355469 68.773438 95.707031 C 68.757812 97.316406 69.070312 98.910156 69.691406 100.382812 C 70.3125 101.859375 71.222656 103.183594 72.371094 104.269531 C 74.617188 106.53125 77.632812 107.792969 80.769531 107.789062 Z M 52.480469 96.628906 L 59.921875 107.503906 L 67.878906 107.503906 L 59.125 94.890625 L 66.898438 83.738281 L 61.738281 80.050781 L 59.460938 83.15625 L 52.472656 93.335938 L 52.472656 72.867188 L 45.507812 67 L 45.507812 107.496094 L 52.472656 107.496094 L 52.472656 96.621094 Z M 81.03125 100.980469 C 79.675781 100.980469 78.371094 100.425781 77.410156 99.429688 C 76.449219 98.4375 75.910156 97.09375 75.910156 95.6875 C 75.910156 94.285156 76.449219 92.9375 77.410156 91.945312 C 78.371094 90.953125 79.675781 90.398438 81.03125 90.398438 L 81.078125 90.398438 C 81.753906 90.398438 82.421875 90.539062 83.039062 90.8125 C 83.660156 91.085938 84.222656 91.488281 84.6875 91.992188 C 85.179688 92.457031 85.570312 93.023438 85.835938 93.65625 C 86.101562 94.289062 86.238281 94.972656 86.234375 95.660156 C 86.214844 97.074219 85.660156 98.421875 84.6875 99.417969 C 83.714844 100.410156 82.40625 100.96875 81.035156 100.972656 Z M 81.03125 100.980469 " />
+  </g>
```

Comment:
This can be simplified a bit by removing the height and width attributes, and removing the group, since it's not used for anything here. Additionally the styles can be converted to attributes, and the color value can be converted into a hexvalue.

Here's a more readable version:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" version="1.1">
  <path fill-rule="nonzero" fill="#7d64ff" d="M 127.828125 128 L 0.171875 128 L 42.667969 34.34375 L 68.269531 53.785156 L 101.660156 0 Z M 81.023438 107.769531 L 81.296875 107.769531 C 84.457031 107.777344 87.496094 106.515625 89.769531 104.25 C 90.9375 103.15625 91.863281 101.820312 92.492188 100.328125 C 93.121094 98.835938 93.433594 97.222656 93.414062 95.59375 C 93.464844 94.027344 93.175781 92.46875 92.570312 91.03125 C 91.960938 89.59375 91.050781 88.3125 89.902344 87.289062 C 87.96875 85.28125 85.359375 84.121094 82.617188 84.046875 L 82.40625 84.046875 C 82.046875 84.042969 81.691406 84.09375 81.347656 84.195312 L 88.085938 73.847656 L 82.71875 69.976562 L 80.175781 73.847656 L 73.683594 84.09375 C 72.566406 85.792969 71.632812 87.289062 71.054688 88.328125 C 70.457031 89.425781 69.941406 90.574219 69.511719 91.757812 C 69.027344 93.011719 68.773438 94.355469 68.773438 95.707031 C 68.757812 97.316406 69.070312 98.910156 69.691406 100.382812 C 70.3125 101.859375 71.222656 103.183594 72.371094 104.269531 C 74.617188 106.53125 77.632812 107.792969 80.769531 107.789062 Z M 52.480469 96.628906 L 59.921875 107.503906 L 67.878906 107.503906 L 59.125 94.890625 L 66.898438 83.738281 L 61.738281 80.050781 L 59.460938 83.15625 L 52.472656 93.335938 L 52.472656 72.867188 L 45.507812 67 L 45.507812 107.496094 L 52.472656 107.496094 L 52.472656 96.621094 Z M 81.03125 100.980469 C 79.675781 100.980469 78.371094 100.425781 77.410156 99.429688 C 76.449219 98.4375 75.910156 97.09375 75.910156 95.6875 C 75.910156 94.285156 76.449219 92.9375 77.410156 91.945312 C 78.371094 90.953125 79.675781 90.398438 81.03125 90.398438 L 81.078125 90.398438 C 81.753906 90.398438 82.421875 90.539062 83.039062 90.8125 C 83.660156 91.085938 84.222656 91.488281 84.6875 91.992188 C 85.179688 92.457031 85.570312 93.023438 85.835938 93.65625 C 86.101562 94.289062 86.238281 94.972656 86.234375 95.660156 C 86.214844 97.074219 85.660156 98.421875 84.6875 99.417969 C 83.714844 100.410156 82.40625 100.96875 81.035156 100.972656 Z M 81.03125 100.980469 " />
```